playground/mocks/server.js

```diff
@@ -67,6 +60,13 @@ function registerService(app, config) {
         res.status(config.status);
       }
     }
+    // Render
```

Comment:
Follow-up for https://github.com/okta/okta-signin-widget/pull/3626 (custom mock server) - status code should be sent before response data